content/governance.md

```diff
@@ -34,19 +34,16 @@ New members may be proposed by any existing member by email to [prometheus-team]
 
 If the new member proposal is accepted, the proposed team member should be contacted privately via email to confirm or deny their acceptance of team membership. This email will also be CC'd to [prometheus-team][team] for record-keeping purposes.
 
-If they choose to accept, the following steps are taken:
-
-* Team members are added to the [GitHub organization][gh] as _Owner_.
-* Team members are added to the [team mailing list][team].
-* Team members are added to the list of team members in this document.
-* New team members are announced on the [developers mailing list][devs] by an existing team member.
+If they choose to accept, the [onboarding](#onboarding) procedure is followed.
 
 Team members may retire at any time by emailing [the team][team].
 
-Team members can be removed by [supermajority vote](#supermajority-vote) on [the team mailing list][team]. For this vote, the member in question is not eligible to vote and does not count towards the quorum.
+Team members can be removed by [supermajority vote](#supermajority-vote) on [the team mailing list][team]. For this vote, the member in question is not eligible to vote, does not count towards the quorum, and may not make any [editorial](#editorial-changes) changes to this and related documents any more.
 
 Upon death of a member, their team membership ends automatically.
 
+In case the member leaves, the [offboarding](#offboarding) procedure is applied.
```

Comment:
Should that include stepping down as maintainer of repos?

Reply:
That is a valid question and should see broader discussion. In the case of forced removal, most certainly.

We always have the mechnanism of team votes which can force anything, including removal as repo maint.

Reply:
Actually, I changed to

`* removed from the [GitHub organization][gh] and related organizations and repositories.
`

which covers pretty much anything. Default to save and re-add as needed.